### PR TITLE
Add modal windows in mobile version

### PR DESCRIPTION
**Pull request by benja27**
--Add modal menu to mobil version

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @benja27,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

You have submitted a project, that has no visible contribution from your pair programming partner. Kindly check Work and Submission mode on the project requirements.
Screenshot 2023-03-07 at 9 23 23 PM



Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: main.js [hunk 1]
```diff
@@ -1,43 +1,42 @@
...
\ No newline at end of file
+});
+
+hamburguerContainer.addEventListener('click', () => {
+  hamburguerIcon.classList.toggle('hide');
+  closeIcon.classList.toggle('show');
+  menuIcon.classList.toggle('show');
+  headerBlur.classList.toggle('blur');
+  logoBlur.classList.toggle('blur');
+});
```
Comment (codecaiine) [inline main.js]: - [ ] Good job so far 👍  But I noticed that your code is quite repetitive 😢 I would like to recommend you write a function that contains a specific logic and then call it multiple times in our code to make your code DRY . You can check the example below ⬇️ 

```
// Mobile Menu Variables
const containerMenu = document.querySelector('.menu');
const closeMenu = document.querySelector('.menu-close');
const openMenu = document.querySelector('.menu-open');
const linkMenu = document.querySelectorAll('.menu a');

// Mobile Menu Functions
function menuOpen() {
  containerMenu.style.display = 'flex';
}
function menuClose() {
  containerMenu.style.display = 'none';
}

// Mobile Menu Events
linkMenu.forEach((link) => {
  link.addEventListener('click', menuClose);
});

openMenu.addEventListener('click', menuOpen);
closeMenu.addEventListener('click', menuClose);
```

File: styles.css [hunk 2]
```diff
@@ -199,44 +199,53 @@ a {
   gap: 2rem;
   font-size: 15px;
   font-weight: 500;
-  width: 0px;
+  width: 0;
 }
-  
-.menu-mobil{
+
+.menu-mobil {
```
Comment (codecaiine) [inline styles.css:205]: - [ ] Good job so far 👍 your menu works as expected but I have a small issue with the design 😢  Please make sure that your mobile menu design is similar to the figma template. Your **Xmark** and menu option spacing are a bit different, please check the screenshots below ⬇️  to see the differences and fix them 👍 

#### You mobile menu design
menuAUTHOR


#### figma template
menuuuuuu

File: README.md [hunk 3]
```diff
@@ -207,6 +207,12 @@ Example:
 - Twitter: [@twitterhandle](https://twitter.com/@rksne123)
 - LinkedIn: [LinkedIn](www.linkedin.com/in/mariano-benjamin-mendez-garcia-a50856177)
 
+👤 **Maedea9**
+
+- GitHub: [@githubhandle](https://github.com/maedea)
```
Comment (codecaiine) [inline README.md:212]: - [ ] [OPTIONAL] I would like to strongly recommend you customize your license by adding you names and application names in order to make it your own and your readme more professional 👍

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @benja27 @Maedea9 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linters errors ✔️ 
- mobile menu works as expected ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @benja27 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**